api/v3/Activity/Getdayswithactivities.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Activity.getdayswithactivities API specification
+ *
+ * @param array $spec description of fields supported by this API call
+ * @return void
+ */
+function _civicrm_api3_activity_getdayswithactivities_spec(&$spec) {
+  $allowed = ['activity_date_time', 'activity_status_id', 'case_id'];
+  $all = civicrm_api3('Activity', 'getfields', array('api_action' => 'get'))['values'];
+
+  $spec = array_filter($all, function ($name) use ($allowed) {
+    return in_array($name, $allowed);
+  }, ARRAY_FILTER_USE_KEY);
+}
+
+/**
+ * Returns list of unique YYYY-MM-DD dates with at least an activity
+ *
+ * @param array $params
+ *   Prameters to be passed to API call to obtain activities list
```

Comment:
Typo in `Prameters`.
Should this be in the same line? In the first function the description is in same line.